ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputFocusEvent.java

```diff
@@ -12,7 +12,7 @@
 import com.facebook.react.uimanager.events.Event;
 import com.facebook.react.uimanager.events.RCTEventEmitter;
 
-/** Event emitted by EditText native view when it receives focus. */
+/** Event emitted by a native view when it receives focus. */
 /* package */ class ReactTextInputFocusEvent extends Event<ReactTextInputFocusEvent> {
```

Comment:
Sorry about these typos - I initially wanted to unify the event classes since they are indeed the same, but was getting build errors in the CI about importing from a different package, even after I made the class public (not an Android dev either), so then changed it back.
I'll give it another try, but it's a little difficult not being able to build master - I'm testing on v63-stable and then need to manually port the changes - hence the typo.

Reply:
Just to clarify - would you like me to make both the view managers for TextInput and View use the same event classes? I.e. merge 4 into 2 or 4 into 1?